### DISCUSSION TITLE
NEW USER - PRINTING - IN PARTICULAR CATEGORIES

### DISCUSSION THREAD

**Discussion by Martinjay00** [moneymanagerex/android-money-manager-ex#1581] [Q&A]
Hi,
If this explanation frustrates you, dear reader, do skip to the end for the question(s).

I'm a new and inexperienced user of the program, using it to track personal expenditure. I'm trying a VERY granular approach when allocating expenditure and so have been  creating loads of categories with sub-categories - but........ This is causing me huge headaches when allocating a category/sub-category to a transaction.

There's a drop down category list from which to select, but because of my decision about having a many category/sub-category structure and entries, the titles in the drop down list a so long that the crucial end to allocate the appropriate category bit can't be viewed and so I'm stumped and frustrated having input all of the categories already, before catching up with transaction entries from 01 Jan 2024.

Would be most grateful for advice and guidance on the following:

Is there an easy way to print out the category structure of my database?

Is there a way for the category dropdown list in the transaction entry window to be made "Re-sizable" using a mouse to see the whole of the individual entries for selection/allocation purposes?

Is there a means of exporting or developing an exporting capability so save the Payee and Category Lists to a separate file in the various, popular formats?

How may I request these as suggestions for future additions to the program, or would they be too difficult?

Most grateful for your advice and comments.

Sincerely,

Martin Parker
Leicester,UK

**Comment by luisdev** [comment]
Can you explain how you're using the categories and sub-categories that require you to have so many of them?

And, can you share a list of some of those categories and sub-categories?

Would using the new Tags feature make it easier for you and reduce the need to have so many categories and sub-categories?

**Comment by wolfsolver** [comment]
Answer n. 2
Take a look to budget functionality. In that screen there are the list of all category/subcategory

Answer n.2b
With desktop version is easy to get full category list
